.github/workflows/repatriate.yml

```diff
@@ -0,0 +1,66 @@
+name: repatriate
+on:
+  push:
+    branches:
+      - release/v*
+
+jobs:
+  repatriate:
+    runs-on: ubuntu-latest
+    name: repatriate
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Log in to GitHub
+        shell: bash
+        run: |
+          git config --global user.name '${{ secrets.GH_AUTO_USER }}'
+          git config --global user.email '${{ secrets.GH_AUTO_EMAIL }}'
+          git config --global 'url.https://${{ secrets.GH_GITHUB_API_KEY }}@github.com.insteadof' https://github.com
+          echo '${{ secrets.GH_GITHUB_API_KEY }}' | gh auth login --with-token
+      - name: Create PR
+        id: step-create-pr
+        shell: bash
+        run: |
+          set -e
+
+          this_ver=${GITHUB_REF##*/}
+          this_branch=release/${this_ver}
+          echo "::set-output this_branch=${this_branch}"
+
+          next_ver=$(git for-each-ref --format='%(refname:lstrip=4)' 'refs/remotes/origin/release/v*' | sort --version-sort | grep -A1 -Fx "$this_ver" | sed 1d)
+          if test -n "$next_ver"; then
+            next_branch="release/${next_ver}"
+          else
+            next_ver="${this_ver%.*}.$((${this_ver##*.}+1))"
+            next_branch='master'
+          fi
```

Comment:
So I had to legit copy and paste crap into the shell to figure out what this does...

...that said, I'm not sure how to make it better. Hmm.